src/document/DocumentAnnotator.ts

```diff
@@ -106,10 +114,15 @@ export default class DocumentAnnotator extends BaseAnnotator {
             return;
         }
 
-        if (mode === Mode.HIGHLIGHT) {
-            this.annotatedEl.classList.add('ba-is-highlighting');
-        } else {
-            this.annotatedEl.classList.remove('ba-is-highlighting');
+        Object.values(ANNOTATION_CLASSES).forEach(className => {
+            if (this.annotatedEl && className) {
```

Comment:
Is `this.annotatedEl` check needed here? It has been checked 5 lines up, right?

Reply:
I see this if I remove that check:
```
Object is possibly 'undefined'.
121                 this.annotatedEl.classList.remove(className);
                    ~~~~~~~~~~~~~~~~
Found 1 error.
```

Reply:
This seems related: https://github.com/microsoft/TypeScript/issues/29916. Could try the `const` assignment

Reply:
Will give this a try.